server/web/setup.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask
+from flask_login import LoginManager
+from models import WebUser
+from app import db
+from .api.register import api
+from bcrypt import gensalt, hashpw
+
+
+def setup(app: Flask):
+    # TODO: Remove
```

Comment:
Are these todos up to date? When is it necessary to remove this?

Reply:
I've moved the test web user to dbsetup